client/app/assets/css/redash.css

```diff
@@ -321,26 +321,44 @@ to add those CSS styles here. */
 }
 
 pivot-table-renderer > table, grid-renderer > div, visualization-renderer > div {
-  overflow: auto;
+  /*overflow: auto;*/
```

Comment:
At this point I assume we can just delete the commented lines, no?

Reply:
If it's not used - yes, it definitely should be removed. But I'll check it to be sure (ATM I don't remember why I commented it out instead of deleting)